libs/subliminal_patch/providers/sucha.py

```diff
@@ -53,41 +53,38 @@ def id(self):
         return self.download_id
 
     def get_matches(self, video):
+        type_ = "episode" if isinstance(video, Episode) else "movie"
         self.found_matches |= guess_matches(
             video,
-            guessit(
-                self.filename,
-                {"type": "episode" if isinstance(video, Episode) else "movie"},
-            ),
+            guessit(self.filename, {"type": type_}),
         )
         self.found_matches |= guess_matches(
             video,
-            guessit(
-                self.guessed_release_info,
-                {"type": "episode" if isinstance(video, Episode) else "movie"},
-            ),
+            guessit(self.guessed_release_info, {"type": type_}),
         )
         return self.found_matches
 
 
 class SuchaProvider(Provider):
     """Sucha Provider"""
-    languages = {Language.fromalpha2(l) for l in ["es"]}
+
+    # This is temporary. Castilian spanish subtitles may exist, but are rare
+    # and currently impossible to guess from the API.
+    languages = {Language("spa", "MX")}
     language_list = list(languages)
     video_types = (Episode, Movie)
 
     def initialize(self):
         self.session = Session()
-        self.session.headers = {
-            "User-Agent": os.environ.get("SZ_USER_AGENT", "Sub-Zero/2")
-        }
```

Comment:
Why did you change the user-agent here?

Reply:
Better for the provider to directly know when a call is from Bazarr. I'm the developer of the Sucha provider btw.

Reply:
I know and that's the reason why we use the SZ_USER_AGENT environment variable here: https://github.com/morpheus65535/bazarr/blob/master/bazarr/init.py#L18-L19

Sub-Zero/2 is only the fallback value if the environment variable isn't set.